Automattic/sensei
# Add featured video to lesson patterns, update pattern looks

Fixes #5787

### Changes proposed in this Pull Request

* Add the Featured Video block to the Video lesson pattern
* Reorder patterns, move video to be the first
* Use example in pattern picker, tweak visuals
* Add featured video example with image
* Update quiz example
* Fix a few unrelated warnings 

### Testing instructions

* Create a new lesson
* In the wizard, go to step 2. Check out the 🔥 patterns
* Pick Video Lesson
* The Featured Video block should be added to the lesson


### Screenshot / Video

<img width="875" alt="image" src="https://user-images.githubusercontent.com/176949/192504283-b0e61b4c-4de0-4b90-b937-fc89d4debb79.png">

<img width="874" alt="image" src="https://user-images.githubusercontent.com/176949/192504330-113fc315-5fa6-424d-a2fa-b1a1be294521.png">
